src/browser-frontend/views/browser/chrome/tabbar/tab.jsx

```diff
@@ -73,6 +79,9 @@ export default class Tab extends PureComponent {
             }),
           } : {}),
         }}
+        draggable="true"
+        onDragStart={this.drag}
+        data={dragging}
```

Comment:
the reason for this was that I needed to be able to select the dragging element, if I add it as a class it gets a random module id added after and I was unable to select it with JS.